instrumentation/graphql/lib/opentelemetry/instrumentation/graphql/tracers/graphql_trace.rb

```diff
@@ -75,10 +75,11 @@ def execute_field(field:, query:, ast_node:, arguments:, object:, &block)
             platform_key = _otel_execute_field_key(field: field)
```

Comment:
Unrelated to this exact diff but is there a reason we can't cache this, like it's done in other fields? I see it depends on `field.trace`, `field.type`, `@trace_scalars`, all of which _look_ stable.

Reply:
That would be better answered by @swalkinshaw or @rmosolgo

Reply:
The actual key lookup is cached within the method, but yeah `trace_field?` is called each time and I think it could be cached 👍

Reply:
While I have your eyes on this @swalkinshaw... would you mind providing a review?